pallets/messages/README.md

```diff
@@ -0,0 +1,65 @@
+# Messages Pallet
+
+Stores block-indexed data for a Schema using `OnChain` or `IPFS` payload location.
+
+## Summary
+
+Messages stores metadata and message payload data for a set Schema.
+Use of the Message pallet payloads is designed for time-centric data.
+Discovery is via the `MessagesInBlock` on new blocks or requesting messages from a block.
+Retrieval is via RPC.
+
+### Metadata vs Payload
+
+Messages have both metadata and payloads.
+The payload should always match the data structure or the message is considered invalid.
+The metadata is the Block Number, Schema Id, and other data useful for discovering and organizing the payload information.
+
+### Payload Options
+
+- `IPFS`: Storage of the CID and length of the file on IPFS
+- `OnChain`: Storage of the entire payload data. Usually for tiny payloads only.
```

Comment:
not a big deal but what does tiny mean?

Reply:
Updated to:
`- `OnChain`: Storage of the entire payload data, usually for sub-256 byte payloads`

Reply:
Merging for now, but let me know if you have thoughts/better ideas and we can tweak it in one of the other docs PRs.